R/aux_functions.R

```diff
@@ -19,17 +23,45 @@
 #'   incubation_period_samples = c(1, 2, 3, 4, 1),
 #'   alpha = 2
 #' )
-incubation_to_generation_time <- function(incubation_period_samples, alpha) {
+incubation_to_generation_time <- function(incubation_period_samples,
+                                          alpha,
+                                          exposure_time = 0,
+                                          latent_period = 0) {
 
   checkmate::assert_numeric(incubation_period_samples, lower = 0, finite = TRUE)
+  checkmate::assert_numeric(exposure_time, lower = 0, finite = TRUE)
   checkmate::assert_number(alpha, finite = TRUE)
+  checkmate::assert_number(latent_period, lower = 0, finite = TRUE)
 
-  out <- sn::rsn(n = length(incubation_period_samples),
-                 xi = incubation_period_samples,
-                 omega = 2,
-                 alpha = alpha)
+  # individual's incubation period delay not absolute time
```

Comment:
this comment could be clearer - but also perhaps can be removed if we change the arguments as I suggested?